src/main/java/teammates/common/datatransfer/attributes/StudentAttributes.java

```diff
@@ -21,73 +23,79 @@
 public class StudentAttributes extends EntityAttributes<CourseStudent> {
 
     // Note: be careful when changing these variables as their names are used in *.json files.
```

Comment:
Not your fault, but let's remove this line; this is accurate for all `*Attributes` and not just this one.